### ISSUE TITLE
[Sample Request] Could add the Unite sample with the generative chat?

### ISSUE THREAD

**Issue by MunteanuAndreiStefan** [Unity-Technologies/sentis-samples#4]
Hi,

I've seen a great sample at Unite. Could you add an example of Generative AI for smart NPCs with voice recognition?

Thanks,
Stefan

**Comment by LiutaurasVysniauskasUnity** [comment]
Thank you for the suggestion! This is known internally as Task 256

**Comment by pauldog** [comment]
Hi we have some new models on [Hugging Face](https://huggingface.co/models?library=unity-sentis&sort=trending). This includes a speech-to-text model (Whisper) and a small language model (Tiny Stories) to get you started.

**Comment by ElectroGamesDev** [comment]
> Hi we have some new models on [Hugging Face](https://huggingface.co/models?library=unity-sentis&sort=trending). This includes a speech-to-text model (Whisper) and a small language model (Tiny Stories) to get you started.

Tiny Stories is used to create stories, so its most likely not helpful with creating smart NPCs.

**Comment by pauldog** [comment]
> > Hi we have some new models on [Hugging Face](https://huggingface.co/models?library=unity-sentis&sort=trending). This includes a speech-to-text model (Whisper) and a small language model (Tiny Stories) to get you started.
> 
> Tiny Stories is used to create stories, so its most likely not helpful with creating smart NPCs.

That's true. Although the same code could be used with other LLMs with very little changes. And there will be some other LLMs up there in the future.

**Comment by ElectroGamesDev** [comment]
> That's true. Although the same code could be used with other LLMs with very little changes. And there will be some other LLMs up there in the future. And some plans are being considered to release the NPC framework. (Not got a timescale on that yet)

How could other LLMs be used with Sentis that aren't in a .sentis file?

**Comment by pauldog** [comment]
[You can convert them](https://docs.unity3d.com/Packages/com.unity.sentis@1.3/manual/serialize-a-model.html)
